specification/redisenterprise/resource-manager/readme.python.md

```diff
@@ -3,19 +3,22 @@
 These settings apply only when `--python` is specified on the command line.
 Please also specify `--python-sdks-folder=<path to the root directory of your azure-sdk-for-python clone>`.
 
-```yaml $(python)
-python:
-  azure-arm: true
-  license-header: MICROSOFT_MIT_NO_VERSION
-  payload-flattening-threshold: 2
-  namespace: Microsoft.Cache
-  package-name: redisenterprise
-  package-version: 2020-10-01-preview
-  clear-output-folder: true
+```yaml $(python) && $(track2)
+python-mode: create
+azure-arm: true
+license-header: MICROSOFT_MIT_NO_VERSION
+namespace: Microsoft.Cache
```

Comment:
@00Kai0 should this extra "namespace: Microsoft.Cache" be removed?  I think we only want the "namespace: azure.mgmt.redisenterprise".

Reply:
Yes, sure. I have removed.